cms/search/tests.py

```diff
@@ -66,3 +66,27 @@ def test_promoted_search(self):
         self.assertContains(response, "Suggested result")
         self.assertContains(response, "Test Bananas Page")
         self.assertContains(response, "Plantains are similar to bananas.")
+
+
+class TestPagination(TestCase):
+    fixtures = ["fixtures/menu-fixtures.json"]
+
+    def test_no_pagination(self):
+        response = self.client.get("/search/?query=no-results")
+        self.assertNotContains(response, "Previous")
+        self.assertNotContains(response, "Next")
+
+    def test_no_prev(self):
+        response = self.client.get("/search/?page=3&query=e")
```

Comment:
Where does the 'no more than three pages' come from, ooi? [non-blocking]

Reply:
I've seeded the DB with pages that only ever return 3 pages of results for `query=e` that way we make sure the next is hidden when we are at the end of a list.